### PR TITLE
Dces 626 add s3 malware protection dev

### PR DESCRIPTION
**Pull request by vvartakMoJ**

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-dces-drc-integration-dev/resources/s3_malware_protection.tf [hunk 1]
```diff
@@ -22,9 +22,15 @@ resource "aws_iam_role" "MemberInfrastructureAccess" {
   })
 }
 
+# Data source for the member-access IAM role
+data "aws_iam_role" "member_access" {
+  depends_on = [aws_iam_role.MemberInfrastructureAccess]
```
Comment (jaskaransarkaria) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-dces-drc-integration-dev/resources/s3_malware_protection.tf:27]: I don't think you need this depends_on

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-dces-drc-integration-dev/resources/s3_malware_protection.tf [hunk 2]
```diff
@@ -43,10 +49,6 @@ resource "aws_guardduty_malware_protection_plan" "malware_protection_plan" {
...
   }
-}
 
-# Data source for the member-access IAM role
-data "aws_iam_role" "member_access" {
-  depends_on = [aws_iam_role.MemberInfrastructureAccess]
-  name = aws_iam_role.MemberInfrastructureAccess.name
-}
\ No newline at end of file
+  depends_on = [data.aws_iam_role.member_access]
```
Comment (jaskaransarkaria) [inline namespaces/live.cloud-platform.service.justice.gov.uk/laa-dces-drc-integration-dev/resources/s3_malware_protection.tf:53]: I don't think you need this depends_on

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-dces-drc-integration-dev/resources/s3_malware_protection.tf (hunk 1 above)
Comment (vvartakMoJ) [reply namespaces/live.cloud-platform.service.justice.gov.uk/laa-dces-drc-integration-dev/resources/s3_malware_protection.tf]: OK, I've removed it now.

File: namespaces/live.cloud-platform.service.justice.gov.uk/laa-dces-drc-integration-dev/resources/s3_malware_protection.tf (hunk 2 above)
Comment (vvartakMoJ) [reply namespaces/live.cloud-platform.service.justice.gov.uk/laa-dces-drc-integration-dev/resources/s3_malware_protection.tf]: OK, I've removed it now.

**Review by jaskaransarkaria — APPROVED** [review]

**Pull request merged**